apigee/adapter.go

```diff
@@ -143,13 +144,12 @@ func (b *builder) Build(context context.Context, env adapter.Env) (adapter.Handl
 	}
 
 	pMan := product.NewManager(*customerBase, env.Logger(), env)
-	if err != nil {
-		return nil, err
-	}
-
 	aMan := auth.NewManager(env)
-	if err != nil {
-		return nil, err
+	var anMan analytics.Manager
+	if b.adapterConfig.UseUap {
```

Comment:
This is ok for now, but we won't keep both implementations.

Reply:
Yep when adding a replacement implementation I prefer to guard behind a flag, that way we don't accidentally break things or become blocked on the new implementation not working properly.